src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,78 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private LinkedList<Entry<K, V>>[] table;
+    private int size;
 
+    public MyHashMap() {
+        table = new LinkedList[INITIAL_CAPACITY];
```

Comment:
The initialization of the `table` array with `new LinkedList[INITIAL_CAPACITY]` will cause an unchecked cast warning because Java does not allow the creation of generic arrays. Consider using `LinkedList<Entry<K, V>>[] table = (LinkedList<Entry<K, V>>[]) new LinkedList<?>[INITIAL_CAPACITY];` to suppress the warning.